app/entities/Product.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Entities;
+
+class Product {
+
+    private $name;
+    private $price;
+    private $taxation;
+
+    /**
+     * Constructor
+     *
+     * @param string $name
+     * @param float $price
+     * @param float $taxation
+     */
+    public function __construct(string $name,float $price,float $taxation) {
```

Comment:
Die Formatierung der Parameter ist etwas eng ;)

Reply:
Wurde angepasst ^^normalerweise integriere ich die "Coding-Rules" in die IDE, so dass die Formatierung sich automatisch anpasst.